frontend/src/components/common/TextEditor/index.tsx

```diff
@@ -0,0 +1,45 @@
+import ReactQuill from 'react-quill-new';
+
+import { ToolbarOptions } from '@customTypes/textEditor';
+import 'react-quill-new/dist/quill.snow.css';
+import './style.css';
+
+interface TextEditorProps {
+  width?: string;
+  height?: string;
+  toolbarOptions?: ToolbarOptions;
+  value: string;
+  onChange: (content: string) => void;
+  onBlur?: () => void;
+  placeholder?: string;
+}
+
+const defaultToolbarOptions: ToolbarOptions = [
+  [{ header: [1, 2, 3, false] }],
+  ['bold', 'italic', 'underline', 'blockquote', 'link'],
+  [{ list: 'ordered' }, { list: 'bullet' }],
+  [{ align: [] }, { color: [] }, { background: [] }],
+  ['clean'],
+];
+
+export default function TextEditor({
+  width = '100%',
+  height = '100%',
+  toolbarOptions = defaultToolbarOptions,
+  value,
+  onChange,
+  onBlur,
+  placeholder,
+}: TextEditorProps) {
+  return (
+    <ReactQuill
+      style={{ width, height }}
+      modules={{ toolbar: toolbarOptions }}
+      theme="snow"
```

Comment:
(정보) theme을 bubble로, readOnly를 true로 하면 스타일을 통일한 채 읽기 전용 모드처럼 볼 수 있다고 합니다.
다만 제가 스토리북에서 실험했을 땐 스타일 적용이 안돼서 적용엔 좀 더 연구가 필요해 보이네요..😅

[출처](https://github.com/zenoamaro/react-quill/issues/567#issuecomment-1451318088)

```suggestion
      theme="bubble"
      readOnly={true}
```

Reply:
유용한 정보 감사합니다! 지원자가 보는 공고 화면과 에디터에서 보여지는 화면의 스타일 일치화는 여전히 필요하겠습니다만, 추후 '미리보기' 등 기능 구현이 필요할 때 아주 유용하게 쓸 수 있을 것 같네요 🙏